RIPS/rip-7560.md

```diff
@@ -67,6 +67,26 @@ smart contracts. This, however, is impossible without an addition of Native Acco
 | MAX_CONTEXT_SIZE       | 65536               |
 | MAX_REVERT_REASON_SIZE | 1024                |
 
+
+### Definitions
+
+* *Transaction*:
+  An action initiated by an account and represented by a set of input parameters.
+  This action will change the state of the Ethereum blockchain when it is included in a block.
```

Comment:
1. we define "EOA Transaction", since we don't care about blobs (they are unmodified)
2. a transaction is a "frame" call from the account to the transaction target (or creating a transaction target)

Reply:
Re 1: Blobs are completely irrelevant to this document. They are still technically EOA transactions, and are not affected. There is no need to mention the blob transaction type, other transaction types, or Santa Claus, in this RIP.
Re 2: It *was* a "frame" but we propose it to be multiple frames. The "Definitions" part does not have a "before" and "after" sections.